src/panels/alarm-table/alarm_details.html

```diff
@@ -106,7 +106,7 @@ <h3>Journal Memo</h3>
                                 <i class="fa fa-external-link"></i>
                             </a>
                         </td>
-                        <td severity>{{summary.reductionKey}}</td>
+                        <td severity>{{summary.label}}</td>
                         <td severity ng-bind-html="summary.description"></td>
```

Comment:
The table header was changed from description to log message, but we still render the description.